cli/chaosli/cmd/root.go

```diff
@@ -22,6 +24,8 @@ import (
 	"github.com/spf13/viper"
 )
 
+var VERSION string
+var APILIBPATH string = fmt.Sprintf("chaosli-api-lib/%v/v1beta1", VERSION)
```

Comment:
A few questions here:
- `API_LIB_PATH` please?
- How is VERSION set?
- Depending on how VERSION is set, aren't we interpolating it into APILIBPATH too soon?

Reply:
- Will edit the variable name
- VERSION is set through the -ldflags, and in practice will be done from the k8s-chaos-ressources Makefile command you commented on earlier, where the release tag is set as VERSION
- very good question, I did not think of kubebuilder versioning/naming while doing this - would love to know what's coming and talk about it very soon

Reply:
The -ldflags can directly set a variable like this? I didn't know that, that's magic and cool, please comment that above VERSION then

Reply:
Fixed